build_sdk.py

```diff
@@ -55,30 +55,29 @@ class ConfigInfo:
 
 SUPPORTED_BOARDS = (
     BoardInfo(
-        name="tqma8xqp1gb",
```

Comment:
It seems there is some rebase quirk here, this overwrites `tqma8xqp1gb` instead of just adding `rockpro64`.